scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/htssop.yaml

```diff
@@ -267,6 +267,53 @@ Texas_PWP_R-PDSO-G20:
     # bottom_pad_size:
     paste_avoid_via: False
 
+Texas_PWP0020T:
+# HTSSOP-20-1EP_4.4x6.5mm_P0.65mm_EP3.4x6.5mm_Mask2.96x2.96mm:
+  size_source: 'https://www.ti.com/lit/ds/symlink/tps2663.pdf#page=49'
+  #                    HTSSOP-20-        1EP_4.4x6.5                        mm_P0.65   mm_EP3.4x6.5                    mm_Mask2.96x2.96mm:
+  custom_name_format: 'HTSSOP-{pincount}-1EP_{size_x:g}x{size_y:g}mm_P{pitch}mm_EP{ep_size_x:g}x{ep_size_y:g}mm_Mask{mask_size_x:g}x{mask_size_y:g}mm{vias:s}'
+  body_size_x:
+    minimum: 4.3
+    maximum: 4.5
+  body_size_y:
+    minimum: 6.4
+    maximum: 6.6
+  overall_size_x:
+    minimum: 6.2
+    maximum: 6.6
+  lead_width:
+    minimum: 0.19
+    maximum: 0.30
+  lead_len:
+    minimum: 0.5
+    maximum: 0.75
+  pitch: 0.65
+  num_pins_x: 0
+  num_pins_y: 10
+  
+  EP_size_x: 3.4
+  EP_size_y: 6.5
```

Comment:
Please use the actual package pad dimensions for `EP_size_[xy]` and EP_size_[xy]_overwrite for the final copper dimensions.